Task7.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+    printf("Enter n to get prime numbers <= n: ");
+    int n = 0;
+    int scanned = scanf("%d", &n);
+    while (scanned == 0)
+    {
+        scanf("%*[^\n]");
+        printf("You didn't enter a number, try again: ");
+        scanned = scanf("%d", &n);
+    }
+    if (n > 1)
```

Comment:
```suggestion
    if (n <= 1)
```
Нужно сначала рассматривать все некорректности ввода, (с `return 0` в конце), а затем уже выполнять основной алгоритм (ветка else будет не нужна, так как в предыдущем коде есть `return`)